angular-docker-sample-app/README.md

```diff
@@ -0,0 +1,75 @@
+# angular-docker-sample-app
+
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 17.1.0.
+
+## Development server
+
+Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The application will automatically reload if you change any of the source files.
+
+## Code scaffolding
+
+Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
+
+## Build
+
+Run `ng build` to build the project. The build artifacts will be stored in the `dist/` directory.
+
+## Running unit tests
+
+Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.io).
+
+## Running end-to-end tests
+
+Run `ng e2e` to execute the end-to-end tests via a platform of your choice. To use this command, you need to first add a package that implements end-to-end testing capabilities.
+
+## Further help
+
+To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI Overview and Command Reference](https://angular.io/cli) page.
+
+## Documentation
+
+- [Development Setup](docs/development-setup.md)
```

Comment:
Development Setup link does not work, leads to 404 - page not found. Can get rid of it if not needed.

Reply:
Will delete this file alltogether and put setup in outer readme